html/style.css

```diff
@@ -135,38 +140,33 @@
 }
```

Comment:
I don't know if there's more style definitions like this, but it seems like this one ends up being unused.

Reply:
Yeah I will be taking a look at which styles arent used anymore, there seems to have been some left over from the old UI.